packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextViewManager.kt

```diff
@@ -30,7 +30,7 @@ import java.util.HashMap
 @Suppress("DEPRECATION")
 @ReactModule(name = ReactTextViewManager.REACT_CLASS)
 @OptIn(UnstableReactNativeAPI::class)
-public class ReactTextViewManager
+public open class ReactTextViewManager
```

Comment:
As you can imagine: we're not fan of this :(

Inheritance makes our code extremely hard to evolve, as every change to a class results in a breaking change.

Ideally the 3p library should use composition to model their behavior around what `ReactTextViewManager` does